src/main/groovy/com/okta/test/mock/tests/ImplicitLocalValidationIT.groovy

```diff
@@ -89,7 +89,7 @@ class ImplicitLocalValidationIT extends ApplicationTestRunner {
             .get("http://localhost:${applicationPort}/api/messages")
         .then()
             .statusCode(200)
-            .body("messages.text", hasItems("I am a robot.", "Hello, word!"))
+            .body("messages.text", Matchers.hasItems("I am a robot.", "Hello, word!"))
```

Comment:
You can import the matcher too:
`import static org.hamcrest.Matchers.hasItems` (something like that anyway)

Reply:
Yes, I don't know which was preferred. Let me static import the method then